gluon/dal.py

```diff
@@ -117,3 +117,11 @@ def _default_validators(db, field):
         DRIVERS['pg8000'] = pg8000
     except:
         pass
+if not DRIVERS.get('sqlite3'):
```

Comment:
Also, I got food poisoning yesterday and don't think I'll be able to make it to class today :(  I'm going to try to do some more work on this however